utils/group-emails-to-lowercase.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python group-emails-to-lowercase.py -group MyConf.org/2017/Reviewers --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# Sets all members that are emails to lowercase.  Notifies user if there are profiles or groups using the
+# uppercase letters in the
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('-g','--group', required=True, help = "the full path of the group ex. MyConf.org/2017")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+def get_profile(value):
+    profile = None
+    try:
+        profile = client.get_profile(value)
+    except openreview.OpenReviewException as e:
+        # throw an error if it is something other than "not found"
+        if e[0][0] != 'Profile not found':
+            print "OpenReviewException: {0}".format(e)
+            raise e
+    return profile
+
+def get_group(value):
+    group = None
```

Comment:
May be we should move this logic to the library, if the profile or group doesn't exist then return None instead of raise an exception